nisystemlink/clients/auth/_auth_client.py

```diff
@@ -0,0 +1,33 @@
+"""Implementation of AuthClient."""
+
+from typing import Optional
+
+from nisystemlink.clients import core
+from nisystemlink.clients.core._uplink._base_client import BaseClient
+from nisystemlink.clients.core._uplink._methods import get
+
+from . import models
+
+
+class AuthClient(BaseClient):
```

Comment:
I think it's very likely we'll want to combine the /niuser and /niauth methods into a single client. There's not really a meaningful distinction between them and they're both implemented in the same service anyway. With that in mind, is `AuthClient` the name we want to go with? I could see `UserClient`, `AdminClient`, `AccessControlClient` off the top of my head.

@spanglerco, @kjohn1922 , @cameronwaterman any opinions?